roles/openshift_bootstrap_autoapprover/tasks/main.yml

```diff
@@ -18,15 +18,58 @@
     value: "{{ osn_image }}"
 
 # TODO: temporary until we fix apply for image stream tags
-- name: Remove the image stream tag
-  command: >
-    {{ openshift_client_binary }} delete -n openshift-infra istag node:v3.10 --ignore-not-found
-    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
-
-- name: Apply the config
-  command: >
-    {{ openshift_client_binary }} apply -f "{{ mktemp.stdout }}"
-    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+- name: Remove the image stream
+  oc_obj:
+    name: "node"
+    kind: ImageStream
+    state: absent
+    namespace: openshift-infra
+
+- name: Create imagestreamtag
+  oc_obj:
+    state: present
+    kind: ImageStreamTag
+    name: "node:v3.10"
```

Comment:
why the colon @vrutkovs  ?

Reply:
That's an ImageStreamTag, created in https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_bootstrap_autoapprover/files/openshift-bootstrap-images.yaml#L4

Reply:
ah okay, thanks